src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -393,22 +395,33 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 		});
 	}
 
+	private _getFontInfo(): ISimpleSuggestWidgetFontInfo {
+		const c = this._terminalConfigurationService.config;
+		const font = this._terminalConfigurationService.getFont(dom.getActiveWindow());
+		const fontInfo: ISimpleSuggestWidgetFontInfo = {
+			fontFamily: font.fontFamily,
+			fontSize: font.fontSize,
+			lineHeight: Math.ceil(1.5 * font.fontSize),
```

Comment:
Does this match what's done for the editor? 1.5* seems random?

Reply:
I did not add this part - just moved it from https://github.com/microsoft/vscode/pull/238593/files#diff-a01baf177fecfd0e1cc7effa04494b16c222ee060128aeef90ff284e3f1640f9L403

Reply:
seems wrong though - should probably terminal line height?

Reply:
i'll investigate as a part of https://github.com/microsoft/vscode/issues/238599